doc/classes/OS.xml

```diff
@@ -734,10 +734,10 @@
 		</method>
 		<method name="request_permission">
 			<return type="bool" />
-			<param index="0" name="name" type="String" />
+			<param index="0" name="type" type="int" enum="OS.PermissionType" />
 			<description>
-				Requests permission from the OS for the given [param name]. Returns [code]true[/code] if the permission has been successfully granted.
-				[b]Note:[/b] This method is currently only implemented on Android, to specifically request permission for [code]"RECORD_AUDIO"[/code] by [code]AudioDriverOpenSL[/code].
+				Requests permission from the OS for the given [param type]. See the [enum PermissionType] constants for available permissions. Returns [code]true[/code] if the permission has been successfully granted.
+				[b]Note:[/b] This method is currently only implemented on Android, to specifically request permission for [code]PERMISSION_RECORD_AUDIO[/code] by [code]AudioDriverOpenSL[/code].
```

Comment:
Question, how should I change here to pass the first check?

BTW, is it still the case as mention in Note? I see that `PermissionUtil` for Android also handles `CAMERA` and `VIBRATE`.